src/affinder/load_tf.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+from magicgui import magic_factory
+from napari.layers import Layer
+
+
+@magic_factory(call_button='Load')
+def load_affine(layer: Layer, affine: str):
+    """Load affine from string.
+
+    Parameters
+    ----------
+    layer : napari.layers.Layer
+        Layer to load affine to.
+    affine : str
+        Affine to load.
+    """
+    affine = np.loadtxt(affine, delimiter=',')
+    layer.affine = affine @ layer.affine
```

Comment:
I actually think this should overwrite the affine on the layer, rather than combine it, but if you really want this option, you could make an extra parameter, e.g. `overwrite: bool = True`, then depending on the parameter you either overwrite or multiply.

Reply:
I think you're right. This should just overwrite the old affine...